code/drasil-lang/Language/Drasil/Expr.hs

```diff
@@ -26,16 +26,23 @@ data BinOp = Frac | Pow | Subt | Eq | NEq | Lt | Gt | LEq | GEq | Impl | Iff | I
   | Dot | Cross
   deriving Eq
 
-data ArithOper = Add | Mul deriving (Eq)
+data ArithOper = Add | Mul
+  deriving Eq
 
-data BoolOper = And | Or deriving (Eq)
+data BoolOper = And | Or
+  deriving Eq
 
 -- | Unary functions
-data UFunc = Norm | Abs | Log | Ln | Sin | Cos | Tan | Sec | Csc | Cot | Arcsin
-  | Arccos | Arctan | Exp | Sqrt | Not | Neg | Dim
+data UFunc = Abs | Log | Ln | Sin | Cos | Tan | Sec | Csc | Cot | Arcsin
```

Comment:
Just to be clear: this kind of tiny step forward is exactly the kind of thing I was thinking of.

Reply:
I made a bit of a mess with the binary functions that I originally attempted, but I think I've got a better idea of how small my steps should be now. Thank you!